### DISCUSSION TITLE
Problem editing module - product page

### DISCUSSION THREAD

**Discussion by MSSDEFENCE** [PrestaShop/PrestaShop#33841] [Q&A]
![Screen Shot 2023-09-05 at 12 05 48](https://github.com/PrestaShop/PrestaShop/assets/144117565/1466225c-68ef-4b08-aeba-346e579ee03c)
Hi I am trying to edit the text on the right column but i am not able to find out how! Any help is welcome!

**Comment by MatShir** [comment]
My guess a custom cms module 🤞 
it doesn't look like a native module
